MSAL/test/automation/tests/MSALBaseUITest.m

```diff
@@ -293,7 +293,10 @@ - (void)closeAuthUIUsingEmbeddedWebview:(BOOL)embedded safariViewController:(BOO
 {
     NSString *buttonTitle = @"Cancel";
 
-    if (safariViewController)
+    BOOL usesSafariAsFallback = ([[[UIDevice currentDevice] systemVersion] floatValue] <= 11.0f
+                                 && !embedded);
+
```

Comment:
Should version check be less than 11, not less than or equal to?

btw, Why not just pass webview type to this method and handle it within?
i.e./
if (webviewtype == sfv ||
    (webviewtype == default && os < 11) buttontitle = done.

Reply:
Done :)